mindjunkies/accounts/tests/test_view.py

```
@@ -1,35 +1,36 @@
 import pytest
-from django.urls import reverse
 from decouple import config
+from django.urls import reverse
 
-from mindjunkies.accounts.models import User
-from mindjunkies.accounts.models import Profile
+from mindjunkies.accounts.models import Profile, User
```

Ruff (0.8.2)

`mindjunkies.accounts.models.Profile` imported but unused

Remove unused import

(F401)